main/guides/subquery-indexing.md

```diff
@@ -0,0 +1,32 @@
+# SubQuery Indexing
+
+This document explains how to index Agoric blockchain data with SubQuery's open source data indexer.
+
+## Overview
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Agoric Cosmos, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
+
+Features include multiple RPC endpoint configurations, multi-worker capabilities and a configurable caching architecture. To find out more, visit our [documentation](https://academy.subquery.network/).
+
+## Getting started
+Take a look at this [SubQuery Starter Project](https://github.com/subquery/cosmos-subql-starter/tree/main/Agoric/agoric-starter) that introduces SubQuery's Agoric support by indexing all transfer events and messages on the Agoric network.
+
+You can also follow along this [step by step guide](https://academy.subquery.network/quickstart/quickstart_chains/cosmos-agoric.html) to get familiar with SubQuery or check out the [Agoric x SubQuery workshop](https://www.youtube.com/watch?v=QC5wQOcWynU) to see an actual demo.
+
+## Running and Hosting your Agoric SubQuery APIs
+SubQuery is open-source, meaning you have the freedom to run it in the following three ways:
+
+* Locally on your own computer, or a cloud provider of your choosing. View the instructions on how to run SubQuery locally [here](https://academy.subquery.network/run_publish/run.html).
+
+* You can publish it to SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/login), where we'll host your SubQuery project in production ready services for mission critical data with zero-downtime blue/green deployments. There even is a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
```

Comment:
I'm adverse to strong claims when writing a guide that we expect to provide as part of Agoric documentation.   Can this paragraph be simplified while still preserving the intent of promoting the managed service?

```suggestion
* You can publish it to SubQuery's [Managed Service](https://managedservice.subquery.network/login), where the SubQuery corporation will host your SubQuery project.  There is also a free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
```